LambdaRuntimeDockerfiles/update-dockerfile.ps1

```diff
@@ -0,0 +1,55 @@
+param ([Parameter(Mandatory)]$Dockerfile)
+
+# Updates the Dockerfile with next ASP.NET Core version and sha256 hash if available
+function Update-Dockerfile ([string]$path) {
+    $nextVersion = Get-NextASPNETVersion -Dockerfile $path
+
+    $shaFilePath = "${nextVersion}-sha.txt"
+
+    Invoke-WebRequest -Uri "https://dotnetcli.blob.core.windows.net/dotnet/checksums/${nextVersion}-sha.txt" -OutFile $shaFilePath
+    if (!$?) {
+        Write-Host "Failed to download checksums for ${nextVersion}. ${nextVersion} is not available yet."
+        return
```

Comment:
Should it return a negative exit code so that the GitHub action fails if this fails to download?

Reply:
Actually not, it will be bad exit if returned non-zero here. The action checks for all the dockerfiles and then make sure there is git diff. 

In case (which will happen mostly) download for checksum fails, action will finish successfully without creating a PR. There are information logs written for debugging purpose.

Doing the way round is going to create a lot of failures which are not actually failures but unavailability of the newer version.

Reply:
My concern is if MS changes the locations of the checksum file or some other assumption made in this PS script how will we notice. Am I missing something?

Reply:
You are right, if the location changes we will never get to know unless someone checks it manually which is rare occasion because we will get to know that there are no auto PRs in last 30 days but that is a lot of cognitive load.

Anyway, I removed the scheduled run and set to manual to keep this action just for updating rather than detecting a new version is available.